pallets/marketplace/src/types.rs

```diff
@@ -80,3 +80,38 @@ where
     /// Reward offered for the job
     pub reward: Reward,
 }
+
+pub type AssetId = u32;
+pub type AssetAmount = u128;
```

Comment:
Is there no way to avoid having concrete types in non benchmarking code, and rather move it to stub.rs that is only included with benchmarking flag ?